### PR TITLE
Credentials input

### PR DESCRIPTION
**Pull request by davyskiba**
This PR implements inputs on the first screen to let users enter their login and password to the backend API. The code was copied from the rider app and adapted to the customer app approach.

Once users enter credentials, they are stored in app private storage and used on further app launches. To clear them, users have to use the "Clear storage" option in the app system settings.

This is the final part of the implementation that will complete the scope of backend integration.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -4,15 +4,22 @@
...
     <string name="order_from_latitude_label">Source latitude</string>
     <string name="order_from_longitude_label">Source longitude</string>
     <string name="order_to_latitude_label">Destination latitude</string>
     <string name="order_to_longitude_label">Destination longitude</string>
     <string name="create_order">Create order</string>
 
-    <string name="trackable_subscription_failed_dialog_message">Subscribing to updates failed, please try again later.</string>
-    <string name="trackable_subscription_failed_dialog_button_text">OK</string>
+    <string name="trackable_subscription_failed_dialog_title">Subscribing to updates failed.</string>
+    <string name="trackable_subscription_failed_dialog_message">{Subscribing to updates failed, p}lease try again later.</string>
```
Comment (KacperKluka) [inline app/src/main/res/values/strings.xml:22]: I think there're some typos - do we need those `{` and `}` in this text? :thinking:

**Review by KacperKluka — CHANGES_REQUESTED** [review]
Looks good except for one thing :wink:

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (davyskiba) [reply app/src/main/res/values/strings.xml]: bd89d0335ed4bf6733b2200360c92800514c76f0

**Review by KacperKluka — APPROVED** [review]
LGTM :+1:

**Pull request merged**